src/views/VendingMachineApp.js

```diff
@@ -0,0 +1,38 @@
+import View from "./View.js"
+import Navigation from "./Navigation.js";
+import ProductManage from "./ProductManage.js";
+import ChargeAmountManage from "./ChargeAmountManage.js";
+import ProductPurchase from "./ProductPurchase.js";
```

Comment:
```suggestion
import "./Navigation.js";
import "./ProductManage.js";
import "./ChargeAmountManage.js";
import "./ProductPurchase.js";
```
VendingMachineApp.js 파일 내에서 import 해온 Class 변수를 사용하고 있지 않습니다. 각 Class를 로드하고 customElements.define() 을 호출하기 위한 용도라면 import 정도만 해도 괜찮을것같은데 어떠신가요?

Reply:
네~ 한번 그렇게 적용해 보겠습니다 👍